spec/helpers/webmock_helpers.rb

```diff
@@ -1,51 +1,129 @@
-def mock_api
-  authorize_stub(Amorail.config.api_endpoint, 
-    Amorail.config.usermail, 
-    Amorail.config.api_key)
+module AmoWebMock
+  def mock_api
+    authorize_stub(
+      Amorail.config.api_endpoint,
+      Amorail.config.usermail,
+      Amorail.config.api_key)
 
-  account_info_stub(Amorail.config.api_endpoint)
-end
+    account_info_stub(Amorail.config.api_endpoint)
+  end
 
+  def authorize_stub(endpoint, usermail, api_key)
+    cookie = 'PHPSESSID=58vorte6dd4t7h6mtuig9l0p50; path=/; domain=amocrm.ru'
+    stub_request(:post, "#{endpoint}/private/api/auth.php?type=json")
+      .with(
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.